app/views/events/index_past.html.erb

```diff
@@ -1,4 +1,6 @@
-<p><%= t('.past_events_thanks') %></p>
+<% provide :title, t('events.index.past_events').titleize %>
```

Comment:
1. see above - keeping locales
2. no .titleize as we want to keep consistency (no "camel case" headlines, first letter should be capitalized and the following lower case)